scripts/pre-upgrade.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
```

Comment:
The script needs to be invoked manually even when we are doing helm upgrade since the helm `pre-upgrade hooks` are executed on an upgrade request after templates are rendered, but before any resources are updated.

Reply:
Ok.